lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +205,12 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    response = requests.get(url=url,
```

Comment:
remember to make timeouts among requests

Reply:
Fixed